src/NUnitFramework/framework/Assume.cs

```diff
@@ -16,7 +16,7 @@ namespace NUnit.Framework
     /// result. If an assumption is not met, the test
     /// should produce an inconclusive result.
     /// </summary>
-    public class Assume
+    public abstract class Assume
```

Comment:
Why `abstract` and not `static`? I don't see anything that makes this `abstract` and it is used as-is? Same with the other usages of `abstract`.

Reply:
Nevermind, I see Joseph's comment. That is a good point.